tic-tac-toe/src/redux/Game/actions.js

```diff
@@ -0,0 +1,15 @@
+export const actions = {
+  CLICK_SQUARE: 'CLICK_SQUARE',
+  MAKE_MOVE: 'MAKE_MOVE'
+};
+
+export const actionsCreators = {
+  clickSquare: values => ({
+    type: actions.CLICK_SQUARE,
+    payload: { history: values.history, stepNumber: values.stepNumber, xIsNext: values.xIsNext }
```

Comment:
You don't need to know the history, stepNumber or who goes next when clicking a square. You should tell the reducer who clicked where, and the reducer should manage those variables.